problems.rb

```diff
@@ -0,0 +1,13 @@
+# TODO: fix this whole function
+def printHiabunchoftimesyay(numberofpotatoess)
+  $i = 0
+  $num = numberofpotatoess
+  while $i < $num  do
+   puts("hi" )
+   $i +=1
+   $p = 0
+   while $p < 3  do
```

Comment:
This is probably not the best way to do this. Less looping, also more comments so I can understand what to heck is happening.